### PR TITLE
Send CORS headers for browser clients using gRPC-web

### LINKED ISSUE #3525: Support CORS for grpc-web / websocket calls
As a developer, I want to be able to import the API dynamically into a script on a different URL. Currently, such http requests are blocked by CORS.

### PR DESCRIPTION
**Pull request by niloc132**
Fixes #3525

### CODE REVIEW

**Review by nbauernfeind — APPROVED** [review]

**Review by mofojed — COMMENTED** [review]
So just allowing all origins, not allowing it to be configured? Seems like it'd be fine since we need to auth anyway.

**Review by vbabich — APPROVED** [review]

File: server/jetty/src/main/java/io/deephaven/server/jetty/JettyBackedGrpcServer.java [hunk 1]
```diff
@@ -96,6 +103,57 @@ public JettyBackedGrpcServer(
         // Add an extra filter to redirect from / to /ide/
         context.addFilter(HomeFilter.class, "/", EnumSet.noneOf(DispatcherType.class));
 
+        // If requested, permit CORS requests
+        FilterHolder holder = new FilterHolder(CrossOriginFilter.class);
```
Comment (devinrsmith) [inline server/jetty/src/main/java/io/deephaven/server/jetty/JettyBackedGrpcServer.java:107]: Are there considerations where one might _not_ want CORS support? If so, we could consider adding an optional configuration piece in `io.deephaven.server.jetty.JettyConfig`.

Comment (niloc132) [reply server/jetty/src/main/java/io/deephaven/server/jetty/JettyBackedGrpcServer.java:107]: Fair, though I can't think of any (we don't use browser cookies for anything here, so a malicious website using a user's browser to make calls on their behalf wouldnt really do anything).

My personal vote is that this land as is, then dig in more with parameterizing all of this for the march release (which DnD is almost inevitably going to move to for this release anyway...).

Comment (devinrsmith) [reply server/jetty/src/main/java/io/deephaven/server/jetty/JettyBackedGrpcServer.java:107]: LGTM

**Pull request merged**